diff --git a/omnibus/config/software/datadog-agent-integrations-py2.rb b/omnibus/config/software/datadog-agent-integrations-py2.rb
index 82541b5406d7..51109f9ff1eb 100644
--- a/omnibus/config/software/datadog-agent-integrations-py2.rb
+++ b/omnibus/config/software/datadog-agent-integrations-py2.rb
@@ -12,7 +12,7 @@
 dependency 'pip2'
 
 
-if arm?
+if arm? && !osx?
   # psycopg2 doesn't come with pre-built wheel on the arm architecture.
   # to compile from source, it requires the `pg_config` executable present on the $PATH
   dependency 'postgresql'
diff --git a/omnibus/config/software/datadog-agent-integrations-py3.rb b/omnibus/config/software/datadog-agent-integrations-py3.rb
index 04bf041707fb..b64b9b0081bc 100644
--- a/omnibus/config/software/datadog-agent-integrations-py3.rb
+++ b/omnibus/config/software/datadog-agent-integrations-py3.rb
@@ -15,7 +15,7 @@
 dependency 'snowflake-connector-python-py3'
 dependency 'confluent-kafka-python'
 
-if arm?
+if arm? && !osx?
   # psycopg2 doesn't come with pre-built wheel on the arm architecture.
   # to compile from source, it requires the `pg_config` executable present on the $PATH
   dependency 'postgresql'
@@ -81,6 +81,9 @@ if osx?
   blacklist_packages.push(/^aerospike==/)
   blacklist_folders.push('aerospike')
   blacklist_folders.push('teradata')
+
+  # Hack
+  blacklist_packages.push(/^pyyaml==/)
 end
 
 if arm?
@@ -233,7 +236,10 @@ build do
     end
 
     # Adding pympler for memory debug purposes
-    requirements.push("pympler==0.7")
+    requirements.push("pympler==0.7\n")
+
+    # Hack
+    requirements.push("pyyaml==6.0.1\n")
 
     # Render the filtered requirements file
     erb source: "static_requirements.txt.erb",
