diff --git a/rtloader/three/CMakeLists.txt b/rtloader/three/CMakeLists.txt
index 93822e6904..5d9ec1f120 100644
--- a/rtloader/three/CMakeLists.txt
+++ b/rtloader/three/CMakeLists.txt
@@ -1,4 +1,5 @@
 cmake_minimum_required(VERSION 3.12)
+set(Python3_EXECUTABLE "/opt/datadog-agent/embedded/bin/python3")
 find_package (Python3 COMPONENTS Interpreter Development)
 
 if(Python3_VERSION_MINOR LESS "7")
