diff --git a/omnibus/config/software/datadog-agent-mac-app.rb b/omnibus/config/software/datadog-agent-mac-app.rb
index d6e7e00d2229..81b02ca9b67c 100644
--- a/omnibus/config/software/datadog-agent-mac-app.rb
+++ b/omnibus/config/software/datadog-agent-mac-app.rb
@@ -20,7 +20,7 @@
 
     # Add swift runtime libs in Frameworks (same thing a full xcode build would do for a GUI app).
     # They are needed for the gui to run on MacOS 10.14.3 and lower
-    command "$(xcrun --find swift-stdlib-tool) --copy --scan-executable \"#{app_temp_dir}/MacOS/gui\" --scan-folder \"#{app_temp_dir}/Frameworks\" --platform macosx --destination \"#{app_temp_dir}/Frameworks\" --strip-bitcode --strip-bitcode-tool $(xcrun --find bitcode_strip)"
+    command "$(xcrun --find swift-stdlib-tool) --copy --scan-executable \"#{app_temp_dir}/MacOS/gui\" --scan-folder \"#{app_temp_dir}/Frameworks\" --platform macosx --destination \"#{app_temp_dir}/Frameworks\" --strip-bitcode"
 
     block do # defer in a block to allow getting the project's build version
       erb source: "Info.plist.erb",
