diff --git a/omnibus/config/software/datadog-agent-integrations-py3.rb b/omnibus/config/software/datadog-agent-integrations-py3.rb
index e3e6a7607a..bf18b9da57 100644
--- a/omnibus/config/software/datadog-agent-integrations-py3.rb
+++ b/omnibus/config/software/datadog-agent-integrations-py3.rb
@@ -81,12 +81,12 @@ build do
     when linux_target?
       arm_target? ? "linux-aarch64_py3.txt" : "linux-x86_64_py3.txt"
     when osx_target?
-      "macos-x86_64_py3.txt"
+      "macos_py3.txt"
     when windows_target?
       "windows-x86_64_py3.txt"
   end
   lockfile = windows_safe_path(project_dir, ".deps", "resolved", lockfile_name)
-  command "#{python} -m pip install --require-hashes --only-binary=:all: --no-deps -r #{lockfile}"
+  command "#{python} -m pip install --require-hashes --no-binary=:all: --no-deps -r #{lockfile}"
 
   # Prepare build env for integrations
   wheel_build_dir = windows_safe_path(project_dir, ".wheels")
